utils/inpsSoapClient.ts

```diff
@@ -159,21 +159,23 @@ export function createClient(endpoint: NonEmptyString): ISoapClientAsync {
             .chain(_ => {
               return ConsultazioneSogliaIndicatoreResponse.decode({
                 ..._,
-                DatiIndicatore: {
-                  ..._.DatiIndicatore,
-                  Componenti: Array.from(
-                    xmlDocument.getElementsByTagNameNS(
-                      INPS_NAMESPACE,
-                      "Componente"
-                    )
-                  ).map(familyMemberElement => ({
-                    CodiceFiscale: familyMemberElement.getAttribute(
-                      "CodiceFiscale"
-                    ),
-                    Cognome: familyMemberElement.getAttribute("Cognome"),
-                    Nome: familyMemberElement.getAttribute("Nome")
-                  }))
-                }
+                DatiIndicatore: _.DatiIndicatore
+                  ? {
```

Comment:
can we have this fixed into another pr ? (so we can merge that before everything else)

Reply:
done here https://github.com/pagopa/io-functions-bonus/pull/55